README.md

```diff
@@ -584,7 +584,7 @@ rate2: squants.market.CurrencyExchangeRate = USD/JPY 100.0
 
 scala> // OR
      | val rate3 = JPY(100) -> USD(1)
-rate3: squants.market.CurrencyExchangeRate = USD/JPY 100.0
+rate3: (squants.market.Money, squants.market.Money) = (100.0 JPY,1.0 USD)
```

Comment:
I traced this to the 0.13.15 upgrade.  Before that commit, it formats as CurrencyExchangeRate.  After the commit, it formats as a two Money's, which is the underlying type.  That commit only includes minor version upgrades of a few libraries.